newsfragments/33136.significant.rst

```diff
@@ -0,0 +1,5 @@
+The "airflow db init", "airflow db upgrade" commands and "[database] load_default_connections" configuration options are deprecated.
```

Comment:
33136 added a significant newsfragment, I think we should add an entry to the `RELEASE_NOTES.rst`

Reply:
Resolved. Thanks!